velox/dwio/parquet/reader/TimestampColumnReader.h

```diff
@@ -109,24 +98,19 @@ class TimestampColumnReader : public IntegerColumnReader {
       // Convert TimestampRange to ParquetInt96TimestampRange.
       ParquetInt96TimestampRange newRange = ParquetInt96TimestampRange(
           range->lower(), range->upper(), range->nullAllowed());
-      this->readWithVisitor(
+      return this->readWithVisitor(
```

Comment:
Do we need 'return' in a void function?

Reply:
No necessary in other case, but for here, it is a trick to reduce the `else` case.